EvalData/models/direct_assessment_document.py

```diff
@@ -496,6 +490,10 @@ class DirectAssessmentDocumentResult(BaseAssessmentResult):
         verbose_name=_('Score'), help_text=_('(value in range=[1,100])')
     )
 
+    mqm = models.TextField(
+        verbose_name=_('MQM'), help_text=_('MQM JSON string')

```

Comment:
It seems that this change requires providing a default value for existing data.

Reply:
The default should be `[]`. What's the proper syntax? `default="[]"` as an argument?

Reply:
Yes, I propose a fix here: https://github.com/AppraiseDev/Appraise/tree/romang/develop-mqm